docs/docsite/rst/dev_guide/module_lifecycle.rst

```diff
@@ -69,13 +69,15 @@ Changing a module or plugin name in the Ansible main repository
 ===============================================================
 
 You can also rename a module and keep a deprecated alias to the old name by using a symlink that starts with _.
-This example allows the ``stat`` module to be called with ``fileinfo``, making the following examples equivalent::
+This example allows the ``stat`` module to be called with ``fileinfo``, making the following examples equivalent:
 
-    EXAMPLES = '''
+For Example:
```

Comment:
```suggestion

```

Reply:
@IMvision12 I think this line is not needed. Could you delete it please?

PR looks good to me otherwise.

Reply:
Yeah Sure